src/app/pages/coursesList/coursesList.template.html

```diff
@@ -1,6 +1,7 @@
 <div class="home-page">
 	<div class="course-list">
-		<course (updateCourse)="sendNewCourseData($event)" (deleteCourse)="askForReals($event)" *ngFor="let course of courseList; let i = index" [course]="course"></course>
+		<course (updateCourse)="sendNewCourseData($event)" (deleteCourse)="askForReals($event)" *ngFor="let course of courseList | orderBy:'startDate'; let i = index" [course]="course"></course>
```

Comment:
It would be more readable if you put each attribute on new line
```jade
<course 
    *ngFor="let course of courseList | orderBy:'startDate'; let i = index"
    [course]="course"
    (updateCourse)="sendNewCourseData($event)"
    (deleteCourse)="askForReals($event)"></course>
```
Or
```jade
<course 
    *ngFor="let course of courseList | orderBy:'startDate'; let i = index"
    [course]="course"
    (updateCourse)="sendNewCourseData($event)"
    (deleteCourse)="askForReals($event)"
></course>
```